python/hyperspace/indexconfig.py

```diff
@@ -1,14 +1,17 @@
-class IndexConfig:
+class CoveringIndexConfig:
     def __init__(self, indexName, indexedColumns, includedColumns):
         """
-        Initializes IndexConfig object.
+        Initializes CoveringIndexConfig object.
         :param indexName: index name
         :param indexedColumns: indexed columns
         :param includedColumns: included columns
-        :return: IndexConfig object
+        :return: CoveringIndexConfig object
 
-        >>> idxConfig = IndexConfig("indexName", ["c1"], ["c2","c3"])
+        >>> idxConfig = CoveringIndexConfig("indexName", ["c1"], ["c2","c3"])
         """
         self.indexName = indexName
         self.indexedColumns = indexedColumns
         self.includedColumns = includedColumns
+
+# TODO(ChungminL): Add deprecation warning
```

Comment:
Will you resolve this soon? Otherwise please create an issue for this and add its link here.
In any case it's not good to add your alias here

Reply:
Actually specifying names for TODO comments is a thing that many developers do for a good reason. To quote the [Google C++ Style Guide](https://google.github.io/styleguide/cppguide.html#TODO_Comments):

> TODOs should include the string TODO in all caps, followed by the name, e-mail address, bug ID, or other identifier of the person or issue with the best context about the problem referenced by the TODO. The main purpose is to have a consistent TODO that can be searched to find out how to get more details upon request. A TODO is not a commitment that the person referenced will fix the problem. Thus when you create a TODO with a name, it is almost always your name that is given.

Reply:
I searched Spark codebase and the first TODO with a name was committed 2 years ago. Let's be consistent with the current TODO policy - an issue link or a follow-up PR soon.

Reply:
I didn't know there is a TODO policy - could you share a link or something?

The issue is created as #477.

Reply:
No documentation yet😅 but recently added TODOs have issue links. (via code review) 

I suggest to add the issue link like below, but I'm fine w/o the comment as the issue itself explains well. 

```
# TODO: Add deprecation warning.
#   See https://github.com/microsoft/hyperspace/issues/477
```